ensime_shared/spec/features/suggestion_format.feature

```diff
@@ -13,22 +13,6 @@ Feature: Format a Completion Suggestion
     | <byname>[ByNameType]   | => ByNameType |
     | <repeated>[VarargType] | VarargType*   |
 
-  Scenario: Format single type parameter
-    Given A list of type parameters:
-      | tparam |
-      | A      |
-    When We concat the type parameters
-    Then We get the format [A]
-
-  Scenario: Format multiple type parameters
-    Given A list of type parameters:
-      | tparam |
-      | A      |
-      | B      |
-      | C      |
-    When We concat the type parameters
-    Then We get the format [A, B, C]
```

Comment:
To be clear, these scenarios actually had nothing to do with the completions menu at all—there is no integration in these tests, they are basically just unit tests for the `concat_tparams` function which is now completely unneeded.